code/handlers/trackservers.q

```diff
@@ -333,13 +333,23 @@ reqprocsnotconn:{[requiredprocs]
     not all requiredprocs in exec proctype from .servers.SERVERS where .dotz.liveh[w]
   }
 
+startupdependent:{ 
+     n:();                                                                                          //variable used to check how many cycles have passed
+     while[.servers.reqprocsnotconn[requiredprocs];
+           n+:1;
+           if[n>cycles;.lg.e[`connectionreport;raze string[process]," cannot connect to ",$[1<count b:((),requiredprocs)except (),exec proctype from .servers.SERVERS where .dotz.liveh[w];","sv string@'b;string[b]]]];
+           .os.sleep[timeintv];
+           .servers.startup[];
+           if[count subscribe;
+              (subscribe[];
+              r:(value subscribe)0)[10=0^type subscribe]]];
```

Comment:
344+345 should be a one-liner + I don't think type can return a null:
```
(subscribe[];r:value[subscribe]0)10=type subscribe
```
This will run `subscribe` regardless of the value from `10=type` is that the intention?
Also place closing brackets on newlines for clarity.